ccd-definition/ComplexTypes/PayingExtraInfo.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "PayingExtraInfo",
+    "ListElementCode": "claimNumberText",
+    "FieldType": "Text",
+    "FieldTypeParameter": "",
```

Comment:
Is this line needed?, with FieldType text, fieldTypeParameter should be ignored anyway